test/helpers/api-integration/api-classes.js

```diff
@@ -59,6 +59,31 @@ export class ApiGroup extends ApiObject {
 
     this._docType = 'groups';
   }
+
+  async addChat (chat) {
+    let group = this;
+
+    if (!chat) {
+      chat = {
+        id: 'Test_ID',
+        text: 'Test message',
+        flagCount: 0,
+        timestamp: Date(),
+        likes: {},
+        flags: {},
+        uuid: group.leader,
+        contributor: {},
+        backer: {},
+        user: group.leader,
+      };
+    }
+
+    let update = { chat };
+
+    this.update(update, '$push');
+
+    return this;
```

Comment:
also update only takes one parameter https://github.com/TheHollidayInn/habitrpg/blob/api-v3-get-group-tests/test/helpers/api-integration/api-classes.js#L20, not an object